clients/feeder/feeder.go

```diff
@@ -127,15 +130,28 @@ func newTestServer(network utils.Network) *httptest.Server {
 		case strings.HasSuffix(r.URL.Path, "get_compiled_class_by_class_hash"):
 			dir = "compiled_class"
 			queryArg = "classHash"
+		case strings.HasSuffix(r.URL.Path, "get_public_key"):
+			dir = "public_key"
+			queryArg = ""
```

Comment:
```suggestion
			dir = "public_key"
			queryArg = "pk"
                        queryMap[queryArg] = queryArg
```
instead of special casing at L144, WDYT?

Reply:
I wanted to keep it consistent with the real endpoint, but this is better to save special case handling at L144.